`golangci-lint run` fails due to SA1019: session.New is deprecated

```
$ golangci-lint run
usn/main.go:33:19: SA1019: session.New is deprecated: Use NewSession functions to create sessions instead. NewSession has the same functionality as New except an error can be returned when the func is called instead of waiting to receive an error until a request is made.  (staticcheck)
        db := dynamo.New(session.New(), &aws.Config{Region: aws.String("ap-northeast-1")})
```

`golangci-lint run` on CircleCI does not fail. I don't know why.

--- a/usn/main.go
+++ b/usn/main.go
@@ -30,7 +30,11 @@ func Handler(ctx context.Context) error {
 		log.Fatal(err)
 	}
 
-	db := dynamo.New(session.New(), &aws.Config{Region: aws.String("ap-northeast-1")})
+	sess, err := session.NewSession()
+	if err != nil {
+		log.Fatal(err)
+	}
+	db := dynamo.New(sess, &aws.Config{Region: aws.String("ap-northeast-1")})
 	table := db.Table(os.Getenv("table"))
 
 	doc.Find("entry").Each(func(_ int, s *goquery.Selection) {

